SKILL.tsv

```diff
@@ -2443,24 +2443,24 @@ SKILL_20150317_002442	$Rush: Stun
 SKILL_20150317_002443	{memo X}Enemies pushed away by [Rush] are inflicted with [Stun] for 2 seconds at a 5% chance per attribute level.
 SKILL_20150317_002444	$Impaler: Double Damage
 SKILL_20150317_002445	The original monster hit by[Impaler] receives double damage.
-SKILL_20150317_002446	Dragging Death: Control ally trap
-SKILL_20150317_002447	[Dragging Death] can drag traps and statues installed by allies.
-SKILL_20150317_002448	Stead Charge: Additional damage against Large monsters
+SKILL_20150317_002446	$Dragging Death: Ally Trap Control
+SKILL_20150317_002447	$Allies can drag prepared traps and statues of [Dragging Death].
+SKILL_20150317_002448	$Steed Charge: Large-type Additional Damage
 SKILL_20150317_002449	When [Stead Charge] is used against a large monster, deals 25% more damage per attribute level
 SKILL_20150317_002450	{memo X}Arrest: Number arrested
 SKILL_20150317_002451	{memo X}Number of monsters hit by[Arrest] increases by (small: 2 per level)(medium 1 per level)
-SKILL_20150317_002452	Repair: Number of times repaired
+SKILL_20150317_002452	$Repair: Repair Count
 SKILL_20150317_002453	{memo X}Use of [Repair] increases by 1 per attribute level.
 SKILL_20150317_002454	{memo X}Weapon TouchUp: retention time
 SKILL_20150317_002455	{memo X}Retention time of [Weapon TouchUp] increases by 10 minutes per attribute level
 SKILL_20150317_002456	{memo X}Armor Touch Up: Retention time
 SKILL_20150317_002457	{memo X}Retention time of [Armor TouchUp] increases by 10 minutes per attribute level
 SKILL_20150317_002458	$Iron Hook: Large-type Monster
-SKILL_20150317_002459	[Iron Hook] prevents large monsters from moving for 0.5 seconds per attribute level.
-SKILL_20150317_002460	Double Weapon Assault: Sub-Weapon damage
-SKILL_20150317_002461	Physical damage of the sub-weapon used in [Double Weapon Assault] increases by 10 per level.
-SKILL_20150317_002462	Jolly Roger: Pirate Flag Defense
-SKILL_20150317_002463	Pirate flag defense of [Jolly Roger] increases by 5 per level.
+SKILL_20150317_002459	$Prevents Large-type monsters from moving for 0.5 second per attribute level with [Iron Hook].
+SKILL_20150317_002460	Double Weapon Assault: Sub-weapon damage
+SKILL_20150317_002461	$Increases the physical damage of the sub-weapon used with [Double Weapon Assault] by 10 per level.
+SKILL_20150317_002462	$Jolly Roger: Pirate Flag Defense
+SKILL_20150317_002463	$Increases the pirate flag's defense of [Jolly Roger] by 5 per level.
```

Comment:
To keep with similar attributes, shouldn't it be:
"Increases the pirate flag's defense increase effect of [Jolly Roger] by 5 per level."